src/fxcorrelator_bengops.py

```diff
@@ -1355,13 +1363,13 @@ def spead_issue_all(corr, beams=[], from_fpga=True):
     :param from_fpga:
     :return:
     """
-    spead_static_meta_issue(corr, beams=beams)
+    spead_static_meta_issue(corr, beams=beams)  # fxcorrelator.spead_issue_meta
     spead_passband_meta_issue(corr, beams=beams)
```

Comment:
Is this one obsolete? It changes passband, centre freq, bandwidth and n_chans. Since we are not using frequency anymore (as we are getting different frequency bands from the f-engine) is this function still needed?